Source/Core/CircleGeometry.js

```diff
@@ -150,9 +147,12 @@ define([
      * Computes the geometric representation of a circle on an ellipsoid, including its vertices, indices, and a bounding sphere.
      *
      * @param {CircleGeometry} circleGeometry A description of the circle.
-     * @returns {Geometry} The computed vertices and indices.
+     * @returns {Geometry|undefined} The computed vertices and indices.
      */
     CircleGeometry.createGeometry = function(circleGeometry) {
+        if (!defined(circleGeometry._ellipseGeometry)) {
+            return;
```

Comment:
This never happens because `_ellipseGeometry` is always defined in the constructor.  Change the constructor so `this._ellipseGeometry` is only set to `new EllipseGeometry(...)` when `radius > 0`